lib/js/belt_Array.js

```diff
@@ -315,6 +315,14 @@ function map(a, f) {
   return mapU(a, Curry.__1(f));
 }
 
+function flatMapU(a, f) {
+  concatMany(map(a, f));
```

Comment:
Looks weird that it doesn't return anything @bobzhang

Reply:
That's my bad. I'll fix that.